app/scripts/clinic-dashboard/controllers/clinic-dashboard.service.js

```diff
@@ -0,0 +1,44 @@
+/*jshint -W003, -W117, -W098, -W026 */
```

Comment:
It holds data necessary for maintaining state and will be used to achieve AA-253 which I have done and AA-251 which am currently doing